.palantir/revapi.yml

```diff
@@ -1254,6 +1254,10 @@ acceptedBreaks:
       new: "method void org.apache.iceberg.data.parquet.BaseParquetWriter<T>::<init>()"
       justification: "Changing deprecated code"
   "1.9.0":
+    org.apache.iceberg:iceberg-api:
```

Comment:
we shouldn't be breaking the API when introducing new methods (see also https://iceberg.apache.org/contribute/#adding-new-functionality-without-breaking-apis). Please add a default impl to the method and remove the API breakage exception from this file